assets/js/theme/common/carousel/utils/arrowAriaLabling.js

```diff
@@ -1,19 +1,19 @@
 import updateTextWithLiveData from './updateTextWithLiveData';
 
-export default ($prevArrow, $nextArrow, actualSlide, actualSlideCount) => {
-    if (actualSlideCount < 2) return;
-    if ($prevArrow.length === 0 || $nextArrow.length === 0) return;
+export default ($prevArrow, $nextArrow, activeSlideIdx, slidesQuantity, { carouselArrowAriaLabel }) => {
```

Comment:
Could we replace destructuring with label param?

Reply:
As we discussed when util needs only one context value destructuring was replaced with relevant data